src/app/pages/arbol-cuentas-contables/cuenta-contable/cuenta-contable.component.html

```diff
@@ -211,10 +211,19 @@
 						</mat-error>
 					</mat-form-field>
 				</mat-grid-tile>
-			</mat-grid-list>
-			<button mat-flat-button
-					color="primary"
-					type="submit">{{ 'CUENTAS_CONTABLES.form_confirmar' | translate }}</button>
-		</form>
+				<mat-grid-tile colspan="6" rowspan="1" *ngIf="action != 'ver'">
+					<button mat-flat-button 						
+						color="primary"
+						type="submit">{{ 'CUENTAS_CONTABLES.form_confirmar' | translate }}</button>				   
+				</mat-grid-tile>
+			</mat-grid-list>			
+		</form>		
+	</nb-card-body>
+</nb-card>
+<nb-card>
+	<nb-card-body>
+			<button mat-flat-button class="btn btn-warning text-white"
+				(click)="openDialog()"
+				type="button">movimientos</button>	
 	</nb-card-body>
 </nb-card>
```

Comment:
(En general) Dejar vacía la última línea de cualquier archivo de texto seguido por Git
```suggestion
</nb-card>

```